core/new-gui/src/app/workspace/service/workflow-status/workflow-status.service.ts

```diff
@@ -31,19 +31,6 @@ export class WorkflowStatusService {
       this.statusSubject.next(event.operatorStatistics);
     });
 
-    this.executeWorkflowService.getExecutionStateStream().subscribe(event => {
-      if (event.current.state === ExecutionState.WaitingToRun) {
-        const initialStatistics: Record<string, OperatorStatistics> = {};
-        this.workflowActionService.getTexeraGraph().getAllOperators().forEach(op => {
-          initialStatistics[op.operatorID] = {
-            operatorState: OperatorState.Initializing,
-            aggregatedInputRowCount: 0,
-            aggregatedOutputRowCount: 0,
-          };
-        });
-        this.statusSubject.next(initialStatistics);
-      }
-    });
```

Comment:
what is this change about?

Reply:
frontend changes all operator's state to `initializing` when user clicks run, including the disabled operators.
originally I thought to exclude the disabled operators, but then I just decided this frontend trick is useless and might be confusing, so I just removed it.